crates/sui-protocol-config/src/lib.rs

```diff
@@ -12,7 +12,7 @@ use tracing::{info, warn};
 
 /// The minimum and maximum protocol versions supported by this build.
 const MIN_PROTOCOL_VERSION: u64 = 1;
-const MAX_PROTOCOL_VERSION: u64 = 35;
+const MAX_PROTOCOL_VERSION: u64 = 36;
```

Comment:
I am ok to not bump this now as we'll not need the properties soon

Reply:
I would vote for not bumping this until its needed/used in the respective environments.